app/qml/form/FeatureFormPage.qml

```diff
@@ -166,6 +167,12 @@ Item {
           onFormStateChanged: featureForm.state = root.formState
         }
 
+        Connections {
```

Comment:
Let's move the entire `Connections` to `FeatureForm.qml` and emit feature form's `saved` signal on success. 

When there was an error in commit let's have a new signal in `FeatureForm`